core/runtime/impl/grandpa_impl.hpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_RUNTIME_IMPL_GRANDPA_IMPL_HPP
+#define KAGOME_CORE_RUNTIME_IMPL_GRANDPA_IMPL_HPP
+
+#include "runtime/grandpa.hpp"
```

Comment:
Only the .cpp's .hpp should be the first include, this does not relate to implementation's .hpp

Reply:
It is main include file anyway, though not mentioned in rules. However, I'm not ready to discuss extension of rules, so I just remove blank line. ok.